src/data-mgt/python/events-monitor/main.py

```diff
@@ -0,0 +1,100 @@
+import json
+import os
+from datetime import datetime, timedelta
+
+import requests
+
+DEVICE_REGISTRY_BASE_URL = os.getenv("DEVICE_REGISTRY_URL", "http://staging-platform.airqo.net/api/v1/")
```

Comment:
We shall revisit this soon. Better you add the recent flag perhaps? More like `recent=yes`? 
**WHY?**
We are about the make the default use-case to be `recent=no` so as to cater for the data scientists....when that change happens as indicated in [this PR](https://github.com/airqo-platform/AirQo-api/pull/453), it will be better to find you ready with `recent=yes` in this job. Thanks.

Reply:
@Baalmart this has been sorted #L63